django/forms/widgets.py

```diff
@@ -5,26 +5,31 @@
 from __future__ import unicode_literals
 
 import copy
+import datetime
 from itertools import chain
+import re
 
 from django.conf import settings
 from django.forms.utils import flatatt, to_current_timezone
+from django.utils import datetime_safe, formats, six
 from django.utils.datastructures import MultiValueDict
-from django.utils.encoding import force_text, python_2_unicode_compatible
+from django.utils.dates import MONTHS
+from django.utils.encoding import force_str, force_text, python_2_unicode_compatible
+from django.utils.formats import get_format
 from django.utils.html import conditional_escape, format_html
-from django.utils.translation import ugettext_lazy
 from django.utils.safestring import mark_safe
-from django.utils import formats, six
+from django.utils.six.moves import range
 from django.utils.six.moves.urllib.parse import urljoin
+from django.utils.translation import ugettext_lazy
 
 __all__ = (
-    'Media', 'MediaDefiningClass', 'Widget', 'TextInput',
-    'EmailInput', 'URLInput', 'NumberInput', 'PasswordInput',
-    'HiddenInput', 'MultipleHiddenInput', 'ClearableFileInput',
-    'FileInput', 'DateInput', 'DateTimeInput', 'TimeInput', 'Textarea', 'CheckboxInput',
-    'Select', 'NullBooleanSelect', 'SelectMultiple', 'RadioSelect',
-    'CheckboxSelectMultiple', 'MultiWidget',
-    'SplitDateTimeWidget', 'SplitHiddenDateTimeWidget',
+    'Media', 'MediaDefiningClass', 'Widget', 'TextInput', 'NumberInput',
```

Comment:
Is there any logic to the new ordering? I mean, I don't see any before either ;) . Shouldn't this just be sorted alphabetically?